src/services/goToDefinition.ts

```diff
@@ -149,6 +149,25 @@ namespace ts.GoToDefinition {
         return getDefinitionFromSymbol(typeChecker, type.symbol, node);
     }
 
+    export function getDefinitionAndBoundSpan(program: Program, sourceFile: SourceFile, position: number): DefinitionInfoAndBoundSpan {
+        const definitions = getDefinitionAtPosition(program, sourceFile, position);
+
+        if (!definitions || definitions.length === 0) {
+            return undefined;
+        }
+
+        // TODO: Add textSpan for triple slash references (file and type).
+        const comment = findReferenceInPosition(sourceFile.referencedFiles, position);
+        if (comment && tryResolveScriptReference(program, sourceFile, comment) || findReferenceInPosition(sourceFile.typeReferenceDirectives, position)) {
+            return { definitions, textSpan: undefined };
+        }

```

Comment:
What does this block do?  Is it for ctrl-click on the file path in a triple-slash reference?  Is the TODO still relevant?

Reply:
This block checks if the position we're searching is within a comment and if that comment is a script reference (triple slash). If true we don't search for a textSpan as the method for that involves a comment parser and complicates a little bit more the process, so undefined is returned instead for now.

The TODO is relevant because of this. Ctrl + Click is going to be enabled with definitions but no textSpan for underlining it is implemented yet.

Reply:
Added bug #19447 for tracking this.